index.html

```diff
@@ -134,7 +134,10 @@
             border-t border-gray-500 text-white text-md 
             flex-col md:flex-row max-w-8xl">
             <div class="mt-1">
-                ©  Copyright 2021 <a href="https://sefglobal.org/" target="_blank">Sustainable Education Foundation</a>
+                ©  Copyright 
+                <script>
+                    document.write(new Date().getFullYear())
+                    </script> <a href="https://sefglobal.org/" target="_blank">Sustainable Education Foundation</a>
```

Comment:
```suggestion
                <script> document.write(new Date().getFullYear()) </script>
                <a href="https://sefglobal.org/" target="_blank">Sustainable Education Foundation</a>
```

@VishwaAmarasinghe A small suggestion

Reply:
Ok brother I will update that